src/services/auth/plugins/password/schemas.ts

```diff
@@ -1,134 +1,109 @@
+import { Type } from '@sinclair/typebox';
 import { StatusCodes } from 'http-status-codes';
 
 import { FastifySchema } from 'fastify';
 
 import { error } from '../../../../schemas/fluent-schema';
 
-export const passwordLogin: FastifySchema = {
+export const passwordLogin = {
   tags: ['password'],
   summary: 'Log in with email and password',
   description:
     'Log in with email and password. The user must provide a valid email, password, and captcha. The captcha is used to prevent brute force attacks.',
-  body: {
-    type: 'object',
-    required: ['email', 'password', 'captcha'],
-    properties: {
-      email: { type: 'string', format: 'email' },
-      password: { type: 'string', format: 'strongPassword' },
-      captcha: { type: 'string' },
-      url: {
-        type: 'string',
-        format: 'uri',
-      },
+  body: Type.Object(
+    {
+      email: Type.String({ format: 'email' }),
+      password: Type.String({ format: 'strongPassword' }),
+      captcha: Type.String(),
+      url: Type.Optional(Type.String({ format: 'uri' })),
     },
-    additionalProperties: false,
-  },
-  querystring: {
-    type: 'object',
-    properties: {
-      lang: { type: 'string' },
+    { additionalProperties: false },
+  ),
+  querystring: Type.Object(
+    {
+      lang: Type.Optional(Type.String()),
     },
-    additionalProperties: false,
-  },
-};
+    { additionalProperties: false },
+  ),
+} as const satisfies FastifySchema;
```

Comment:
I see not all schemas are annotated with this, is it necessary, or can it be omitted ?

Reply:
It can be omitted, but I think it is a good practice to mention it, so we have autocompletion when editing the schema. Also there was some schema that has the properties `query` instead of `querystring`...